nvflare/fuel/utils/function_utils.py

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import importlib
+import inspect
+from typing import Callable
+
+
+def find_task_fn(task_fn_path: str) -> Callable:
+    """
+    Find and return a callable task function based on its module path.
+
+    Parameters:
+    - task_fn_path (str): The path to the task function in the format "module_path.function_name".
+
+    Returns:
+    - Callable: The callable task function.
+    """
```

Comment:
Thanks for all the docstrings, it helps a lot.

Our other docstrings are following google python docstring, I think we should be consistent throughout the code base. (google python doc: https://google.github.io/styleguide/pyguide.html)

For example this should be:

```
"""Finds and returns a callable task function based on its module path.
    Args:
        task_fn_path (str): The path to the task function in the format "module_path.function_name".
    Returns:
        Callable: The callable task function.
"""
```